### PR TITLE
fix: assign query and body to params

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,12 +29,10 @@ module.exports = function (app, translate) {
     }
 
     if (!params) {
-      params = this.request.body
```
Comment (haoxins) [inline index.js:32]: when we using `koa-bodyparser`, `this.request.body` is always `non null Object` ~

Comment (dead-horse) [reply index.js:32]: maybe we should detect the method and only use `this.query` when request method is `GET`?I don't want to mixed `query` and `body` here.

Comment (haoxins) [reply index.js:32]: DONE :)

**Review by dead-horse — APPROVED** [review]

**Comment by dead-horse** [comment]
3.0.1

File: index.js [hunk 2]
```diff
@@ -29,9 +29,9 @@ module.exports = function (app, translate) {
...
-        ? this.request.body
-        : this.query;
+      params = ['GET', 'HEAD'].includes(this.method.toUpperCase())
```
Comment (fengmk2) [inline index.js:32]: method already be upper case in node http server. https://github.com/nodejs/node/blob/a577bde917c5cbc20c15972b9fe50a8c0adf5e94/lib/_http_server.js#L111

**Pull request merged**